docs/table.rst

```diff
@@ -53,7 +53,17 @@ From Parsons Table
       - Write a table to a local html file
     * - :py:meth:`~parsons.etl.tofrom.ToFrom.to_dataframe`
       - Pandas Dataframe [1]_
-      - Return a Pandas dataframe 
+      - Return a Pandas dataframe
+    * - :py:meth:`~parsons.etl.tofrom.ToFrom.to_append_csv`
```

Comment:
I'm not sure why this has a different format than all the others, but the actual method name is `append_csv`, not `to_append_csv`. Can you make that change?

Reply:
ohh surely.